test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -308,78 +301,6 @@ namespace {
 grpc::testing::ApiFuzzer* g_api_fuzzer = nullptr;
 }
 
-////////////////////////////////////////////////////////////////////////////////
-// client connection
-
-static void sched_connect(grpc_closure* closure, grpc_endpoint** ep,
-                          gpr_timespec deadline);
-
-typedef struct {
-  grpc_closure* closure;
-  grpc_endpoint** ep;
-  gpr_timespec deadline;
-} future_connect;
-
-static void do_connect(future_connect fc) {
-  if (g_api_fuzzer->Server() != nullptr) {
-    grpc_endpoint* client;
-    grpc_endpoint* server;
-    grpc_passthru_endpoint_create(&client, &server, nullptr, true);
-    *fc.ep = client;
-    start_scheduling_grpc_passthru_endpoint_channel_effects(
-        client, g_api_fuzzer->ChannelActions());
```

Comment:
By deleting this we are making ChannelActions a no-op i.e we loose the ability to schedule arbitrary time-varying  network throughput. Can you add a TODO for me so that we can incorporate it into fuzzing event engine ?

Reply:
or for the time being maybe these api_fuzzer deletions can be reverted ?

Reply:
so with the removal of the vtable here I think we're using fuzzing event engine for this job... maybe it's clearer if I just pull passthru_endpoint out entirely?